Python/cp/Number_of_K-Length_Sublists_with_Average_at_Least _Target.py

```diff
@@ -0,0 +1,72 @@
+# Number of K-Length Sublists with Average at Least Target
+
+# Given a list of integers nums, and integers k and target
+# return the number of sublists whose length is k and its average value ≥ target.
+
+# Constraints:-
+
+# n ≤ 100,000 where n is the length of nums
+
+# Approach : First i will take the sum of [0:k] elements and check it's avg.
+# If avg greater than target ans count is increased
+# then in the loop starting from kth index adding the new element 
+# and removeing the initial added elements simultaneously
+
+
```

Comment:
Great work! Make a small change by mentioning the time complexity at last; after the last line of code. 
Happy learning!

Reply:
@bhav09 sir after approach i have mentioned ? should i change?

Reply:
Just make sure the complexity and I/O is written after the code.

Reply:
ok sir

Reply:
@bhav09 sir as you said changes have been made to the file, please review sir